src/style/load_sprite.js

```diff
@@ -42,10 +42,10 @@ export default function(baseURL: string,
             const result = {};
 
             for (const id in json) {
-                const {width, height, x, y, sdf, pixelRatio} = json[id];
+                const {width, height, x, y, sdf, pixelRatio, stretchX, stretchY, content} = json[id];
```

Comment:
I know we don't have validation for the existing properties either, but I think it'd be good to add validation so that we catch errors early before they go into placement.

Reply:
added validation in 3e1038c6cf5c574d30a1c3fdf55abfedb9a70660